cfgov/cfgov/settings/base.py

```diff
@@ -0,0 +1,91 @@
+import os
+from unipath import Path
+
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+SITE_ROOT = Path(__file__).ancestor(5).child('cfgov-refresh')
+GRUNT_WATCH = []
```

Comment:
@anselmbradford actually, i'm going to remove it entirely just reviewed the code again and its unnecessary